### DISCUSSION TITLE
Migration guide from go-structured-query to sq

### DISCUSSION THREAD

**Discussion by vincentdaniel** [bokwoon95/sq#2] [Ideas]
As [go-structured-query](https://github.com/bokwoon95/go-structured-query) is no longer active and sq is now recommended, it would be awesome for the documentation to include a migration guide so that teams could easily migrate to its spiritual successor!

**Comment by bokwoon95** [comment]
Good idea, I'll make a migration guide for moving to sq 👍

**Comment by bokwoon95** [comment]
The migration guide is live at [https://bokwoon.neocities.org/sq#migrating-from-go-structured-query](https://bokwoon.neocities.org/sq#migrating-from-go-structured-query), thanks for the suggestion

**Reply by vincentdaniel** [reply]
Awesome, thanks a lot! It will be super helpful! 👍
